### DISCUSSION TITLE
Incremental Static Regeneration (ISR) no regenerate the page content using revalidate

### DISCUSSION THREAD

**Discussion by gpolanco** [vercel/next.js#32959] [Help]
I have the same problem with the version 12.0.7.

I use [Catch all routes](https://nextjs.org/docs/routing/dynamic-routes#catch-all-routes) in the File **pages/[[slug]].tsx**

I request the data from an API made with Strapi, this is the way I do the generation of the pages

## getStaticPaths
```ts
export const getStaticPaths: GetStaticPaths = async ({ defaultLocale }) => {
	const paths = await getAllPagePaths();

	return {
		fallback: 'blocking',
		paths: paths.map((route) => {
			const routePath = route.locale === defaultLocale ? route.path : route.path.substring(3);

			return {
				locale: route.locale,
				params: {
					path: routePath.split('/').filter((part) => part),
				},
			};
		}),
	};
};
```
## getStaticProps
```ts
export const getStaticProps: GetStaticProps<Props, IStaticPathParams> = async (context) => {
	const { params, locale, defaultLocale } = context;
	const path = getFormatPath(locale, params);

	const content: IPageContent | null = await pageContentGetPageByUrl(path);

	if (!content) {
		return {
			notFound: true,
			revalidate: Number(process.env.NEXT_REVALIDATE_TIME || 10),
		};
	}

	return {
		props: {
			content,
			locale,
			path,
		},
		revalidate: Number(process.env.NEXT_REVALIDATE_TIME || 10),
	};
};
```

I have read the documentation and I see that I just need to add the `revalidate` property in **getStaticProps** and the` fallback` property in **getStaticPaths**, this should work.

Any idea what it could be? Has anyone been able to solve this problem?

**Comment by alexandrosk** [comment]
Same for me, no idea why, I would love to have an answer for this.

**Comment by gpolanco** [comment]
I have updated to the latest version of nextjs, and it seems to work fine. I have also disabled the cache I had from Cloudflare.
I think the problem was external cache to nextjs
